src/main/model.py

```diff
@@ -155,11 +157,16 @@ def load_models(self):
                 model_name = json_object[name_path_key] if name_path_key in json_object else str(
                     config_file_path.parent).replace(self.path, '')
 
-            sentence_transformer_model = SentenceTransformer(model_path, device=self.device)
-
-            pipelines[model_name] = sentence_transformer_model
-            print(f"Model loaded: $model_name")
-        return pipelines
+                try:
+                    sentence_transformer_model = SentenceTransformer(model_path, device=self.device)
+                    pipelines[model_name] = sentence_transformer_model
+                    if default_pipeline is None:
+                        default_pipeline = sentence_transformer_model
+                    print(f"Model loaded: {model_name}")
+                except:
+                    print(f"Could load mode: {model_name}. Skipping it.")
```

Comment:
Message should say: "couldn't load model .... " can the error be printed with this message?

Reply:
I think you added that message. Do you want to change it?